cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/polling/ClouderaManagerPollingServiceProvider.java

```diff
@@ -77,6 +80,13 @@ public PollingResult startPollingCmStartup(Stack stack, ApiClient apiClient) {
                 new ClouderaManagerStartupListenerTask(clouderaManagerApiPojoFactory, clusterEventService));
     }
 
+    public PollingResult startPollingCmHostStatusHealthy(Stack stack, ApiClient apiClient, Set<String> hostnamesToWaitFor) {
+        LOGGER.debug("Waiting for Cloudera Manager hosts to connect and become healthy. NodeCount={}, Nodes=[{}], [Server address: {}]",
+                hostnamesToWaitFor.size(), hostnamesToWaitFor, stack.getClusterManagerIp());
+        return pollCommandWithTimeListener(stack, apiClient, null, POLL_FOR_5_MINUTES,
```

Comment:
I made a change to use api checker instead of command checker if there is no CM command, be aware of the status of this PR: https://github.com/hortonworks/cloudbreak/pull/12188

Reply:
if you want, we can delay my PR to not cause extra work on this PR by causing conflicts in the code

Reply:
Thanks. Would be fantastic if we could delay it a bit. That looks like a bunch of merge conflicts coming up.

Reply:
ok, sure, I put a do_not_merge label onto my PR